Sources/PaltaLib/Services/Configuration/ConfigurationService.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+final class ConfigurationService {
+    
+    private let networkService: NetworkServiceInterface
+    
+    init(networkService: NetworkServiceInterface) {
+        self.networkService = networkService
+    }
+    
+    public func requestConfigs(apiKey: String, completion: @escaping (Result<Config, Error>) -> Void) {
```

Comment:
It seems that configs are re-fetched each SDK start that is great for development environment but that might lead to problems in the production env. I suggest adding some kind of persistent cache with TTL about 12 hours and update config only when cache is outdated.